lib/dependabot/file_parsers/java/maven.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "nokogiri"
+
+require "dependabot/dependency"
+require "dependabot/file_parsers/base"
+
+module Dependabot
+  module FileParsers
+    module Java
+      class Maven < Dependabot::FileParsers::Base
+        def parse
+          doc = Nokogiri::XML(pom.content)
+          doc.css("dependencies dependency").map do |dependency_node|
+            Dependency.new(
+              name: dependency_name(dependency_node),
+              version: dependency_node.at_css("version").content,
+              package_manager: "maven",
+              requirements: [{
+                requirement: dependency_node.at_css("version").content,
+                file: "pom.xml",
+                groups: [],
+                source: nil
+              }]
+            )
+          end
+        end
+
+        private
+
+        def dependency_name(dependency_node)
+          [
+            dependency_node.at_css("groupId").content,
+            dependency_node.at_css("artifactId").content
+          ].join("/")
```

Comment:
I don't know if this is very important, but usually `:` is used as a separator in various documentation and tooling such as Maven CLI output and Gradle. E.g. `com.google:guava`. When in context of a dependency with a version, it's specified as `com.google:guava:23.3-jre`

Reply:
Useful! I was wondering about that!